test/spec/grammar/validator_spec.js

```diff
@@ -0,0 +1,40 @@
+const expect = require('chai').expect;
+const parse = require('../../../lib/dsl/api').parse;
+
+describe('The JDL Validator', () => {
```

Comment:
There's an actual class for this file, so I'd rather use `JDLSyntaxValidatorVisitor` as provided.